src/watchmaker/static/config.yaml

```
@@ -47,4 +47,4 @@ linux:
 windows:
   - salt:
       salt_debug_log: null
-      installer_url: https://watchmaker.cloudarmor.io/repo/saltstack/salt/windows/Salt-Minion-3003.3-Py3-AMD64-Setup.exe
+      installer_url: https://watchmaker.cloudarmor.io/watchmaker/repo/saltstack/salt/windows/Salt-Minion-3004-2-Py3-AMD64-Setup.exe
```

while this works, prefer leaving out the leading "watchmaker" prefix

```suggestion
      installer_url: https://watchmaker.cloudarmor.io/repo/saltstack/salt/windows/Salt-Minion-3004-2-Py3-AMD64-Setup.exe
```